config/jobs/gardener/gardener-apidiff-master.yaml

```diff
@@ -0,0 +1,18 @@
+presubmits:
+  gardener/gardener:
+  - name: pull-gardener-apidiff-master
+    cluster: gardener-prow-build
+    skip_if_only_changed: "^docs/|\\.md$"
+    decorate: true
+    optional: true
+    branches:
+    - ^master$ # run on master branch
+    path_alias: gardener/gardener
```

Comment:
Shouldn't be needed as our modules are identified via default pattern `github.com/org/repo`.